### DISCUSSION TITLE
Imagemagick commands takes too much server load and CPU usage

### DISCUSSION THREAD

**Discussion by atfsolace** [ImageMagick/ImageMagick#2554] [Help]
Hi 
Some Imagemagick commands take to much CPU usage.
what is the reason of this? and how to resolved this CPU usage issue for Imagemagick commands?

![Screenshot_3](https://user-images.githubusercontent.com/70207338/92991431-5f3fa380-f501-11ea-8546-519c73ac37dc.png)

Please help for this.

Thank you.

**Comment by snibgo** [comment]
Does "404h" mean 404 hours, or what?

What are the commands?

What version of IM, on what platform?

What problem do you want to solve? The total time taken, or the CPU utilisation, or what?

**Reply by atfsolace** [reply]
Thank you for your reply

Yes, 404h means 404 hours

 Version: ImageMagick 6.9.3-8 Q16 x86_64 2019-07-23 http://www.imagemagick.org
 Features: Cipher DPC OpenMP
 Delegates (built-in): bzlib cairo fontconfig freetype jng jp2 jpeg lcms png rsvg tiff x zlib

On Linux Platform. 

I want to solved the problem of command total time taken or executing from 404 hours and CPU load is almost 104/103

Please check below command:
'env LC_CTYPE=en_US.UTF-8 printf "♥ ♠ testing testsss ♥ ♠" |  /usr/bin/convert -background "white" -size 223.89703225806x38.277419354839 -fill "#000000" -font fonts/ttf/ArialNarrow.ttf -gravity center label:@- /userUploads/9743406/20584975279.jpg'

Please suggest any solution.

Thank you.

**Comment by snibgo** [comment]
Simplifying your command ...
```
$ printf "hearts spades testing testsss hearts spades" | convert -background "white" -size 223.89703225806x38.277419354839 -fill "#000000" -gravity center label:@- 20584975279.jpg
```
...and running it on my laptop (under Cygwin bash) takes less than one second.

I don't know what "env LC_CTYPE=en_US.UTF-8" at the start of your command is supposed to do.

If it takes 404 hours for you (does it work after that?), clearly something is badly wrong.

**Reply by atfsolace** [reply]
running it on my laptop (under Cygwin bash) takes less than one second.
  >> Running on my laptop on localhost its takes less time as well for me.
 

If it takes 404 hours for you (does it work after that?), clearly something is badly wrong.
 >> 404 hours takes on the server and its executed 

> env LC_CTYPE=en_US.UTF-8 printf "♥ ♠ testing testsss ♥ ♠" | /usr/bin/convert -background "white" -size 223.89703225806x38.277419354839 -fill "#000000" -font fonts/ttf/ArialNarrow.ttf -gravity center label:@- /userUploads/9743406/20584975279.jpg

> I am using this command for copy paste dynamic miscellaneous symbols likes hearts shape diamonds shape etc, below is the output of this command
![20584975279](https://user-images.githubusercontent.com/70207338/93099325-83e47880-f6c5-11ea-94d7-5516b8180024.jpg)

But not finding any solution for this

Thanks.
